src/App.tsx

```diff
@@ -58,5 +63,10 @@ export const App = () => {
     }
   }, [isNativeOnboardOpen, openNativeOnboard, showWelcomeModal])
 
-  return <Routes />
+  return (
+    <>
+      {showConsentBanner && isConsentBannerEnabled && !isMobileApp && <ConsentBanner />}
```

Comment:
Fine that way, but would be more declarative to move the `isConsentBannerEnabled` and `!isMobileApp` checks to `selectShowConsentBanner`